src/RobotMap.cpp

```diff
@@ -1,99 +1,105 @@
+std::shared_ptr<Relay> RobotMap::lightsLeftUnderGlow;
+std::shared_ptr<Relay> RobotMap::lightsRightUnderGlow;
 
 void RobotMap::init() {
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+
     LiveWindow *lw = LiveWindow::GetInstance();
 
-    elevatorhaySqueeze.reset(new DoubleSolenoid(0, 1, 6));
-    lw->AddActuator("Elevator", "haySqueeze", elevatorhaySqueeze);
-    
-    elevatorrightHaySqueezeSensor.reset(new DigitalInput(2));
-    lw->AddSensor("Elevator", "rightHaySqueezeSensor", elevatorrightHaySqueezeSensor);
-    
-    elevatorleftHaySqueezeSensor.reset(new DigitalInput(3));
-    lw->AddSensor("Elevator", "leftHaySqueezeSensor", elevatorleftHaySqueezeSensor);
-    
-    elevatorbrake.reset(new DoubleSolenoid(0, 2, 5));
-    lw->AddActuator("Elevator", "brake", elevatorbrake);
-    
-    elevatorleftSpool.reset(new CANTalon(2));
-    lw->AddActuator("Elevator", "leftSpool", elevatorleftSpool);
-    
-    elevatorrightSpool.reset(new CANTalon(4));
-    lw->AddActuator("Elevator", "rightSpool", elevatorrightSpool);
-    
-    chassisleftLightSensor.reset(new DigitalInput(0));
-    lw->AddSensor("Chassis", "leftLightSensor", chassisleftLightSensor);
-    
-    chassisrightLightSensor.reset(new DigitalInput(1));
-    lw->AddSensor("Chassis", "rightLightSensor", chassisrightLightSensor);
-    
-    chassisleftFrontWheel.reset(new CANTalon(5));
-    lw->AddActuator("Chassis", "leftFrontWheel", chassisleftFrontWheel);
-    
-    chassisleftRearWheel.reset(new CANTalon(11));
-    lw->AddActuator("Chassis", "leftRearWheel", chassisleftRearWheel);
+//..........Talons..........
+
+    //..........chassis..........
+
+    chassisLeftFrontWheel.reset(new CANTalon(kLeftFrontID));
+    lw->AddActuator("Chassis", "LeftFrontWheel", chassisLeftFrontWheel);
+
+    chassisLeftRearWheel.reset(new CANTalon(kLeftRearID));
+    lw->AddActuator("Chassis", "leftRearWheel", chassisLeftRearWheel);
+
+    chassisRightFrontWheel.reset(new CANTalon(kRightFrontID));
+    lw->AddActuator("Chassis", "rightFrontWheel", chassisRightFrontWheel);
+
+    chassisRightRearWheel.reset(new CANTalon(kRightRearID));
+    lw->AddActuator("Chassis", "rightRearWheel", chassisRightRearWheel);
+
+    //..........elevator..........
+
+    elevatorLeftSpool.reset(new CANTalon(kLeftSpoolID));
+    lw->AddActuator("Elevator", "leftSpool", elevatorLeftSpool);
+
+    elevatorRightSpool.reset(new CANTalon(kRightSpoolID));
+    lw->AddActuator("Elevator", "rightSpool", elevatorRightSpool);
+
+    //..........intake..........
+
+    intakeLeftSpinner.reset(new CANTalon(kLeftSpinnerID));
+    lw->AddActuator("Intake", "leftSpinner", intakeLeftSpinner);
+
+    intakeRightSpinner.reset(new CANTalon(kRightSpinnerID));
+    lw->AddActuator("Intake", "rightSpinner", intakeRightSpinner);
+
+//..........solenoids..........
+
+    //..........elevator..........
+
+    elevatorHaySqueeze.reset(new DoubleSolenoid(0, kHaySqueezeF, kHaySqueezeR));
```

Comment:
I mean this one is more acceptable, the magic number, as we only use one PCM... buuuutt we could still give it a const. (It refers to the PCM id in case you were wondering)